frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -76,6 +78,21 @@ const validateUserTemplate: VmSettingsValidator = (field, options) => {
   const { getState, id } = options;
   const state = getState();
 
+  const forcedSingleUserTemplateName = iGetCommonData(state, id, VMWizardProps.userTemplateName);
+  const userTemplateInitialized = iGetVmSettingAttribute(
+    state,
+    id,
+    VMSettingsField.USER_TEMPLATE,
+    'initialized',
```

Comment:
we don't even need initialized here - lets just check loaded property of VMWizardProps.userTemplates